tests/ui/not_send_auto_trait2.stderr

```diff
@@ -0,0 +1,22 @@
+error[E0277]: the trait bound `PyObject: Ungil` is not satisfied in `[closure@$DIR/tests/ui/not_send_auto_trait2.rs:8:26: 10:10]`
+   --> tests/ui/not_send_auto_trait2.rs:8:12
+    |
+8   |           py.allow_threads(|| {
+    |  ____________^^^^^^^^^^^^^_-
+    | |            |
+    | |            within `[closure@$DIR/tests/ui/not_send_auto_trait2.rs:8:26: 10:10]`, the trait `Ungil` is not implemented for `PyObject`
+9   | |             println!("{:?}", string);
+10  | |         });
+    | |_________- within this `[closure@$DIR/tests/ui/not_send_auto_trait2.rs:8:26: 10:10]`
+    |
+    = note: required because it appears within the type `UnsafeCell<PyObject>`
+    = note: required because it appears within the type `PyAny`
+    = note: required because it appears within the type `PyString`
```

Comment:
It may be nice to add `!Ungil` implementation to one of the macros in `src/types/mod.rs` so that the error stops here at `PyString`.

Reply:
Fair point. The implementation of `!Ungil for PyAny` as suggested by my other comment would be enough anyway.